src/helpers/setup/GatewayHelper.js

```diff
@@ -1,191 +1,27 @@
 'use strict';
 
-const Web3 = require('web3');
 const AbiBinProvider = require('../../AbiBinProvider');
-const CoGatewayHelper = require('./CoGatewayHelper');
+const {
+  sendTransaction,
+  deprecationNoticeChainSetup,
+} = require('../../utils/Utils');
+const EIP20Gateway = require('../../ContractInteract/EIP20Gateway');
+const EIP20CoGateway = require('../../ContractInteract/EIP20CoGateway');
```

Comment:
`EIP20CoGateway` is not used.
Similarly `ContractName` is not used in this file.

Reply:
`txOptions` is not being used in any xxxRawTx methods.